tests/pubsub/testtimedcache.nim

```diff
@@ -33,6 +35,23 @@ suite "TimedCache":
       3 notin cache
       4 in cache
 
+    check:
+      cache.del(4).isSome()
+      4 notin cache
+
     check:
       not cache.put(100, now + 100.seconds) # expires everything
       100 in cache
+      2 notin cache
+
+  test "spam":
```

Comment:
Maybe sth like "can handle a large number of entries"? If that's the purpose of this test. Is there any reason why it starts at 101?

Reply:
no particular reason except to stress the hashset a bit more (by not starting at 0)

Reply:
6c42af5f0108b16e85f876ff8c13b7d5f03703db